File: 3656-minimum-number-of-operations-to-make-elements-in-array-distinct/3656-minimum-number-of-operations-to-make-elements-in-array-distinct.cpp
class Solution {
public:
   int minimumOperations(vector<int>& nums) {
    vector<int> freq(101, 0);
    for(int i = nums.size()-1; i >= 0; --i){
        if(++freq[nums[i]] > 1) return ceil((double)(i + 1)/3);
    }
    return 0;
}
};

File: 3656-minimum-number-of-operations-to-make-elements-in-array-distinct/3656-minimum-number-of-operations-to-make-elements-in-array-distinct.java
class Solution {
    int minimumOperations(int[] nums) {
    int[] freq = new int[101];
    Arrays.fill(freq, 0);
    for(int i = nums.length-1; i >= 0; --i){
        if(++freq[nums[i]] > 1) return (int)Math.ceil((double)(i + 1)/3);
    }
    return 0;
}
}